packages/frontend-plugin-api/src/blueprints/SignInPageBlueprint.test.tsx

```diff
@@ -60,20 +62,11 @@ describe('SignInPageBlueprint', () => {
 
     const tester = createExtensionTester(extension);
 
-    expect(tester.data(SignInPageBlueprint.dataRefs.component)).toBeDefined();
+    const Element = tester.get(SignInPageBlueprint.dataRefs.component)!;
```

Comment:
I would suggest renaming to `Component`, otherwise, it feels like I should be using `tester.reactElement` instead.
```suggestion
    const Component = tester.get(SignInPageBlueprint.dataRefs.component)!;
```